doc/user_guide/axes.rst

```diff
@@ -404,17 +417,61 @@ method:
 .. code-block:: python
 
     >>> axis.get_axis_dictionary()
-    {'name': <undefined>,
-    'units': <undefined>,
-    'navigate': <undefined>,
-    'size': 20,
-    'scale': 0.5,
-    'offset': 10.0}
+    {'_type': 'UniformDataAxis',
+     'name': <undefined>,
+     'units': <undefined>,
+     'navigate': <undefined>,
+     'size': 20,
+     'scale': 0.5,
+     'offset': 10.0}
 
 This dictionary can be used, for example, in the :ref:`initilization of a new
 signal<signal_initialization>`.
 
 
+Adding/Removing axes to/from a signal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Usually, the axes are directly added to a signal during :ref:`signal 
+initialization<signal_initialization>`. However, you may wish to add/remove
+axes from the `AxesManager` of a signal.
+
+Note that there is currently no consistency check whether a signal object has
+the right number of axes of the right dimensions. Most functions will however
+fail if you pass a signal object where the axes do not match the data 
+dimensions and shape.
+
+**Add a single axis** to the `AxesManager` of a signal using ``_axes.append()``:
+
+.. code-block:: python
+    
+    >>> s.axes_manager._axes.append(evaxis)
```

Comment:
Only public API should be documented in the user guide.

Reply:
OK, I guess it is enough that `axes_manager.create_axes` can append a list of axes dictionaries or axes objects, so we do not need to have an extra public function for single axes.

However, the `axes_manager.create_axes` is in fact rather an `append_axes` function and might be confused with the `axes.create_axis` function. On the other hand, I would refrain from renaming such a public function not to break any scripts.